src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,81 @@
+package duke.tasks;
+
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    /**
+     * Constructs a <code>Task</code> object.
+     * @param description The description of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false; // Initially, the task is not done
+    }
+
+    /**
+     * Marks the task as done.
+     */
+    public void markDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * Marks the task as not done.
+     */
+    public void unmarkDone() {
+        this.isDone = false;
+    }
+
+    /**
+     * Returns the completion status of the task.
+     * @return The completion status of the task.
+     */
+    public boolean getIsDone() {
+        return this.isDone;
+    }
+
+    /**
+     * Returns the description of the task.
+     * @return The description of the task.
+     */
```

Comment:
Should this formatting for javadocs be changed? (from my recollection, there should be a space between description and return statement)

Reply:
Yes, there should be a space, mistake by me